client/landing/stepper/declarative-flow/copy-site.tsx

```diff
@@ -52,7 +52,7 @@ const copySite: Flow = {
 						{ ...props }
 						title={ translate( 'We’re copying your site' ) }
 						subtitle={ translate(
-							'This may take a few minutes. Feel free to close this window and we will email you when the process is complete.'
+							'This may take a few minutes. Feel free to close this window, we’ll email you when it’ done.'
```

Comment:
The 's' is missing.

Reply:
Oops, true, I missed that 🙈

Reply:
Fixed! https://github.com/Automattic/wp-calypso/pull/72324/commits/8f48a1da069e67d27d0a7c09f2d645fd117dd00e

You would be a good player of https://10typosgame.com/